proto/feeburner/params.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+package neutron.feeburner;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/neutron-org/neutron/x/feeburner/types";
+
+// Params defines the parameters for the module.
+message Params {
+  option (gogoproto.goproto_stringer) = false;
+  // Defines Neutron denom
+  string neutron_denom = 1;
+  // Defines Gaia denom
+  string gaia_denom = 2;
```

Comment:
why do you need gaia denom here?

Reply:
Because it is an IBC denom and it should be configured? or can we just calculate it in advance and set it in genesis, too?

Reply:
No, not sure that we can predefine it in the genesis. But what is the reason of having it in the params?  Because it is possible to have any coin as fee coin (osmo, juno, xprt etc) and not only atom. So we need to have other approach or to have list of 2rd party denoms here.

Reply:
I think we can simplify by saying that all NTRN tokens are burned, and all non-NTRN tokens are sent to treasury. @foxpy can you implement this?

Reply:
> all NTRN tokens are burned

that means `NeutronDenom` param should be left intact.

> all non-NTRN tokens are sent to treasury

I think I can implement that using `ViewKeeper`'s `GetAllBalances` function, of course if it does what I expect it to.